test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -335,4 +335,63 @@ def add_locations
     assert_response :success
     assert_equal 1, location.reload.location_parameters.count
   end
+
+  test "should create with comma separated name" do
+    name = "#{RFauxFactory.gen_alpha}, #{RFauxFactory.gen_alpha}"
+    post :create, params: { :location => { :name => name } }
+    assert_response :success, "creation with name #{name} failed with code #{response.code}"
+    show_response = JSON.parse(@response.body)
+    refute_empty show_response
```

Comment:
if you parse the response, it would be good to verify it contains the right name too, or remove the JSON.parse line if we don't use it later.